aries-site/src/examples/templates/forms/FilterExample.js

```diff
@@ -97,25 +94,13 @@ export const FilterExample = () => {
               />
             </FormField>
             {serverTypes && (
-              <FormField label="HPE Server Types">
-                <Box gap="small" pad="xsmall" round="xsmall">
-                  {serverTypes.map(server => {
-                    return (
-                      <CheckBox key={server} name={server} label={server} />
-                    );
-                  })}
-                </Box>
+              <FormField label="HPE Server Types" htmlFor="server-types">
```

Comment:
Is this FilterExample being consumed anywhere? If not, should we delete?

Reply:
it's on the Forms templates page